src/main/java/com/exasol/bucketfs/client/PrintExceptionMessageHandler.java

```diff
@@ -11,7 +13,12 @@ public class PrintExceptionMessageHandler implements IExecutionExceptionHandler
     @Override
     public int handleExecutionException(final Exception exception, final CommandLine commandLine,
             final ParseResult parseResult) throws Exception {
-        commandLine.getErr().println(commandLine.getColorScheme().errorText(exception.getMessage()));
+        if (!Objects.isNull(exception.getCause())) {
+            commandLine.getErr().println(commandLine.getColorScheme()
+                    .errorText(exception.getMessage() + ". Cause: " + exception.getCause().getMessage()));
```

Comment:
```suggestion
                    .errorText(exception.toString() + ". Cause: " + exception.getCause().toString()));
```

Reply:
Also print exception class. This will help debugging.
Maybe even print the complete stack trace.

Reply:
Not the expected format. I'll leave it like this for now.